tutordiscovery/templates/discovery/build/discovery/Dockerfile

```diff
@@ -16,7 +16,7 @@ RUN useradd --home-dir /openedx --create-home --shell /bin/bash --uid ${APP_USER
 USER ${APP_USER_ID}
 
 ###### Git-clone course-discovery repo
-ARG DISCOVERY_REPOSITORY=https://github.com/edx/course-discovery.git
+ARG DISCOVERY_REPOSITORY='{{ COURSE_DISCOVERY_REPOSITORY }}'
```

Comment:
Isn't this line causing `tutor config save` to crash? The setting you introduced in plugin.py will be prefixed by `DISCOVERY_`, causing the setting to be named `DISCOVERY_COURSE_DISCOVERY_REPOSITORY`.

Reply:
OHkk Nice catch. Let me rename it to `DISCOVERY_REPOSITORY`.